[MT][browser] `System.Transactions.Tests.AsyncTransactionScopeTests` error: Thread failed to start

Failing after enabling all MT library tests (https://github.com/dotnet/runtime/pull/91536).
```
  [ERROR] FATAL UNHANDLED EXCEPTION: System.Threading.ThreadStartException: Thread failed to start.
    ---> System.ExecutionEngineException: mono_thread_platform_create_thread() failed
    --- End of inner exception stack trace ---
    at System.Threading.Thread.ThrowThreadStartException(Exception ex)
    at System.Threading.Thread.StartCore()
    at System.Threading.Thread.Start(Boolean captureContext, Boolean internalThread)
    at System.Threading.Thread.UnsafeStart()
    at System.Threading.WebWorkerEventLoop.StartExitable(Thread thread, Boolean captureContext)
    at System.Threading.PortableThreadPool.WorkerThread.CreateWorkerThread()
    at System.Threading.PortableThreadPool.WorkerThread.MaybeAddWorkingWorker(PortableThreadPool threadPoolInstance)
    at System.Threading.PortableThreadPool.AdjustMaxWorkersActive()
    at System.Threading.PortableThreadPool.NotifyWorkItemProgress(Object threadLocalCompletionCountObject, Int32 currentTimeMs)
    at System.Threading.PortableThreadPool.NotifyWorkItemComplete(Object threadLocalCompletionCountObject, Int32 currentTimeMs)
    at System.Threading.ThreadPoolWorkQueue.Dispatch()
    at System.Threading.PortableThreadPool.WorkerThread.WorkLoopSemaphoreSuccess(LowLevelLifoAsyncWaitSemaphore semaphore, Object stateObject)
    at System.Threading.LowLevelLifoAsyncWaitSemaphore.InternalAsyncWaitSuccess(LowLevelLifoAsyncWaitSemaphore self, WaitEntry internalWaitEntry)
    at System.Threading.LowLevelLifoAsyncWaitSemaphore.SuccessCallback(IntPtr lifoSemaphore, IntPtr userData)
```
Test got blocked.

--- a/src/libraries/System.Runtime/tests/System.Runtime.Extensions.Tests/System/Random.cs
+++ b/src/libraries/System.Runtime/tests/System.Runtime.Extensions.Tests/System/Random.cs
@@ -516,7 +516,6 @@ namespace System.Tests
         }
 
         [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsThreadingSupported))]
-        [ActiveIssue("https://github.com/dotnet/runtime/issues/91538", typeof(PlatformDetection), nameof(PlatformDetection.IsWasmThreadingSupported))]
         public void Shared_ParallelUsage()
         {
             using var barrier = new Barrier(2);

--- a/src/mono/browser/runtime/loader/config.ts
+++ b/src/mono/browser/runtime/loader/config.ts
@@ -189,7 +189,7 @@ export function normalizeConfig() {
     }
 
     if (MonoWasmThreads && !Number.isInteger(config.pthreadPoolSize)) {
-        // ActiveIssue https://github.com/dotnet/runtime/issues/91538
+        // ActiveIssue https://github.com/dotnet/runtime/issues/75602
         config.pthreadPoolSize = 7;
     }
 

